Taskfile.yml

```diff
@@ -407,87 +398,112 @@ tasks:
 
   docker-init:
     cmds:
-      - docker buildx create --driver=docker-container --name=ferretdb --use=false
+      - docker buildx create --driver=docker-container --name=ferretdb --bootstrap=true --use=false
+      - docker buildx ls
 
-  docker-local:
-    desc: "Build `ferretdb-local` Docker image"
+  docker-build:
     deps: [gen-version]
     cmds:
       - >
         docker buildx build --builder=ferretdb
-        --build-arg VERSION={{.VERSION}}
-        --build-arg COMMIT={{.COMMIT}}
-        --build-arg RACEFLAG={{.RACEFLAG}}
-        --tag=ferretdb-local
-        --load .
+        --file=build/docker/{{.FILE}}.Dockerfile
+        --build-arg=LABEL_VERSION={{.VERSION}}
+        --build-arg=LABEL_COMMIT={{.COMMIT}}
+        --platform={{default "linux" .PLATFORM}}
+        {{range splitList "," .DOCKER_IMAGES}}--tag={{trim .}} {{end -}}
+        --output={{.OUTPUT}}
+        .
     vars:
       VERSION:
         sh: cat build/version/version.txt
       COMMIT:
         sh: cat build/version/commit.txt
 
+  # do not change that target much; see "Building and packaging" in README.md
+  build-release:
+    desc: "Build bin/ferretdb (production build)"
+    deps: [gen-version]
+    cmds:
+      - echo 'bin' > build/version/package.txt
+      - task: docker-build
+        vars:
+          FILE: production
+          PLATFORM: linux/amd64 # TODO https://github.com/FerretDB/FerretDB/issues/2171
+          DOCKER_IMAGES: dummy
```

Comment:
`DOCKER_IMAGES` is a required parameter to prevent mistakes with `docker-*-push` tasks, but in those cases (`type=local`) it is actually unused

PTAL

Reply:
Thanks for the explanation :pray: